hal/halx86/include/halp.h

```diff
@@ -588,6 +588,25 @@ HalInitializeBios(
 #define KiEnterInterruptTrap(TrapFrame) /* We do all neccessary in asm code */
 #endif // _M_AMD64
 
+#ifdef _MINIHAL_
+#if defined(_M_IX86) || defined(_M_AMD64)
```

Comment:
Is it possible to use ddk/ioaccess.h here? (for the non-"BUFFER" functions)

Reply:
Done.